Model/LdapUser.php

```diff
@@ -0,0 +1,316 @@
+<?php
```

Comment:
Hmm good point, some methods deleted. But for Password it's a little bit tricky because it seems that for fosuser the password variable must be overridden? Done in LdapManager:hydrate line 92. So we have a compatibility think here.